### PR TITLE
Bug fix for capture size and some python3 compatibility changes

### PR DESCRIPTION
**Pull request by thouis**
If you want me to split these out, I can.

### CODE REVIEW

File: AlphaGo/util.py [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import sgf
-import string
 from AlphaGo import go
 
+# for board location indexing
+letters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZ'
```
Comment (wrongu) [inline AlphaGo/util.py]: we've been following the convention of capitalizing constants, i.e. `LETTERS`

File: AlphaGo/go.py [hunk 2]
```diff
@@ -87,7 +87,7 @@ def _neighbors(self, position):
 		the given (x,y) position. Basically it handles edges and corners.
 		"""
 		(x, y) = position
-		return filter(self._on_board, [(x - 1, y), (x + 1, y), (x, y - 1), (x, y + 1)])
+		return [xy for xy in [(x - 1, y), (x + 1, y), (x, y - 1), (x, y + 1)] if self._on_board(xy)]
```
Comment (wrongu) [inline AlphaGo/go.py]: maybe make this change in `_diagonals` as well or not at all?

Comment (wrongu) [inline AlphaGo/go.py]: maybe make this change in `_diagonals` as well or not at all?

**Comment by wrongu** [comment]
I think it makes sense to split out into 2 PRs. In terms of Python3 compatibility (which hadn't been a concern, as you can probably tell!) some other files would need to be changed as well

**Comment by thouis** [comment]
Will do.

**Comment by thouis** [comment]
split into #85 and #86

**Pull request closed without merging**